### DISCUSSION TITLE
请问开发者，能否方便对逆时针手式截图增加个截图保存的功能

### DISCUSSION THREAD

**Discussion by andydznj** [shuax/MouseInc.Settings#179] [Ideas]
方便截图图片的直接保存？

**Comment by shuax** [comment]
https://docs.shuax.com/MouseInc/#/faq#%E8%B4%B4%E5%9B%BE%E5%8F%AF%E4%BB%A5%E5%90%8C%E6%97%B6%E4%BF%9D%E5%AD%98%E5%88%B0%E5%89%AA%E8%B4%B4%E6%9D%BF%E5%90%97%EF%BC%9F

**Comment by andydznj** [comment]
谢谢大佬啊，成功了哦
----- 原始邮件 -----
发件人：shuax ***@***.***>
收件人："shuax/MouseInc.Settings" ***@***.***>
抄送人：andydznj ***@***.***>,  Author ***@***.***>
主题：Re:_[shuax/MouseInc.Settings]_请问开发者，能否方便对逆时针手式截图增加个截图保存的功能_(Discussion_#179)
日期：2021年10月16日 13点00分


https://docs.shuax.com/MouseInc/#/faq#%E8%B4%B4%E5%9B%BE%E5%8F%AF%E4%BB%A5%E5%90%8C%E6%97%B6%E4%BF%9D%E5%AD%98%E5%88%B0%E5%89%AA%E8%B4%B4%E6%9D%BF%E5%90%97%EF%BC%9F

&mdash;
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
